examples/demo_rcv/src/main.rs

```diff
@@ -282,7 +286,7 @@ fn handle_input(dmx_recv: &mut SacnReceiver) -> Result<bool> {
                             dmx_recv.set_process_preview_data(v);
                         },
                         Err(_e) => {
-                            bail!(std::io::Error::new(
+                            return Err(std::io::Error::new(
                                 std::io::ErrorKind::InvalidInput, "Preview flag option not 'true'/'false' or otherwise parsable as boolean"));
                         }
```

Comment:
Add a new error to handle places where we do std::io::Error::new() - anything coming from the sacn library should produce an sacn error, anything coming from underheath should be wrapped